.changeset/loud-toes-cheer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@web/test-runner-selenium': patch
```

Comment:
```suggestion
'@web/test-runner-selenium': patch
'@web/test-runner-browserstack': patch
'@web/test-runner-saucelabs': patch
```

Reply:
I just force pushed the PR to add another changeset 🙈